spikeinterface/comparison/collisioncomparison.py

```diff
@@ -42,8 +42,8 @@ def get_label_for_collision(self, gt_unit_id1, gt_unit_id2):
         mask = (self.collision_events['unit_id1'] == gt_unit_id1) & (self.collision_events['unit_id2'] == gt_unit_id2)
         event = self.collision_events[mask]
         
-        score_label1 = self._labels_st1[gt_unit_id1][event['index1']]
-        score_label2 = self._labels_st1[gt_unit_id2][event['index2']]
+        score_label1 = self._labels_st1[self.unit1_ids[gt_index1]][event['index1']]
+        score_label2 = self._labels_st1[self.unit1_ids[gt_index2]][event['index2']]
```

Comment:
More difficult to read no ?

Reply:
Yes, I'll revert that. I was just reading everything to be sure I understood it properly.